Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -429,11 +429,11 @@ int CruRawReader::processHalfCRU(int cruhbfstartoffset)
   uint32_t cruwordsread = 9;
   //reject halfcru if it starts with padding words.
   //this should only hit that instance where the cru payload is a "blank event" of o2::trd::constants::CRUPADDING32
-  if (mHBFPayload[cruhbfstartoffset] == o2::trd::constants::CRUPADDING32 && mHBFPayload[cruhbfstartoffset + 1] == o2::trd::constants::CRUPADDING32) {
+  if (mHBFPayload[cruhbfstartoffset] == o2::trd::constants::CRUPADDING32) { //} && mHBFPayload[cruhbfstartoffset + 1] == o2::trd::constants::CRUPADDING32) {
     if (mVerbose) {
       LOG(info) << "blank rdh payload data at " << cruhbfstartoffset << ": 0x " << std::hex << mHBFPayload[cruhbfstartoffset] << " and 0x" << mHBFPayload[cruhbfstartoffset + 1];
     }
-    mHBFoffset32 += 2;
+    mHBFoffset32++; //= 2;
```

Comment:
Is this correct?

Reply:
yip only reading the first 0xeeeeeeee now not demanding 2, but about to change.